### ISSUE TITLE
I built it successfully, but I can't find the file upload location.

### ISSUE THREAD

**Issue by wonkyungup** [samuelmeuli/action-electron-builder#97]
![스크린샷 2023-08-30 오후 4 56 43](https://github.com/samuelmeuli/action-electron-builder/assets/34180230/4dd29668-9559-473c-bb1c-e6c8220253b7)

It's not in the release, where is it?

```javascript
name: Build/release

on: push

jobs:
  release:
    runs-on: ${{ matrix.os }}

    strategy:
      matrix:
        os: [macos-latest, windows-latest]

    steps:
      - name: Check out Git repository
        uses: actions/checkout@v2

      - name: Install Node.js, NPM and Yarn
        uses: actions/setup-node@v2
        with:
          node-version: 16

      - name: Prepare for app notarization (macOS)
        if: startsWith(matrix.os, 'macos')
        run: |
          mkdir -p ~/private_keys/
          echo '${{ secrets.api_key }}' > ~/private_keys/AuthKey_${{ secrets.api_key_id }}.p8

      - name: Build/release Electron app
        uses: samuelmeuli/action-electron-builder@v1
        with:
          github_token: ${{ secrets.github_token }}
          mac_certs: ${{ secrets.mac_certs }}
          mac_certs_password: ${{ secrets.mac_certs_password }}

          release: ${{ startsWith(github.ref, 'refs/tags/v') }}
        env:
          API_KEY_ID: ${{ secrets.api_key_id }}
          API_KEY_ISSUER_ID: ${{ secrets.api_key_issuer_id }}
```

**Comment by kennethlynne** [comment]
Did you figure it out? 🙏

**Comment by NiceAesth** [comment]
Looking at the logs, it silently fails with the following:
```
  • creating GitHub release  reason=release doesn't exist tag=v1.3.0 version=1.3.0
```
It would seem that something has changed and it will no longer create a release if it does not already exist.

**Comment by wonkyungup** [comment]
@kennethlynne 
No. I'll wait and see, and if it doesn't work, I'll consider something else. If I find a way, I'll share it.

**Comment by TravisBumgarner** [comment]
Dealing with the same thing. I look at the workflow runs I see

```
v1.0.5
Build/release #13: Commit abc123 pushed by travisbumgarner
13 hours ago  3m 18s v1.0.5
```

```
v1.0.5
Build/release #12: Commit abc123 pushed by travisbumgarner
13 hours ago  10m 23s main
```
The workflow with main succeeds. If I look at the logs it looks like everything goes successfully.


The workflow with tag v1.0.5 fails
```
 &#10799; snapcraft is not installed, please: sudo snap install snapcraft --classic  
  &#10799; .../app-builder process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
```
So I'm currently a little confused as to why there are two workflows and why is one failing and one not.

Codebase is here for reference: http://github.com/TravisBumgarner/Todo-Today

**Comment by gugiserman** [comment]
Same case here, also building for macos and windows. Everything green, but can't find the artifcats uploaded.

**Comment by gugiserman** [comment]
Got it.

So, considering you have the default action configuration which will make releases for pushed tags that start with `v`, you want to do the following:

1- Make sure you have committed your latest changes, then create a new tag, for example `git tag v0.1.0`.
2- Push everything to the remote branch, including your newly created tag: `git push origin <branch> --tags` or just that one tag `git push origin v0.1.0`
3- Workflows will run and generate a draft of a release for that tag you pushed, with all your binaries attached. From there you can generate an actual release from the draft and/or simply download the files you need.

My mistake was thinking that I needed to create both a release and a tag for it to run. I plan on changing this in the future so the artifacts get attached to the created release instead. Not sure if possible, but would be much more intuitive for me personally.

<img width="414" alt="image" src="https://github.com/samuelmeuli/action-electron-builder/assets/4623123/5d31e3d0-7b9f-4233-a997-887401c2df1d">

**Comment by Ladvace** [comment]
same problem here, not sure if I didn't follow @gugiserman properly but I still run into this issue
